### ISSUE TITLE
The auto docking function is not working

### ISSUE THREAD

**Issue by Phymin** [automaticaddison/two_wheeled_robot#3]
Hi 

I follow the tutorial https://automaticaddison.com/auto-docking-to-recharge-battery-aruco-marker-and-ros-2/, but the auto docking function is not working as described, the issue is after run the low_battery_pub.py script, the robot will not plan a path to the staging area, but still moves to the previous goal, am I missing some step here? 

BTW, I am using ros2 humble

**Comment by automaticaddison** [comment]
Thanks. Check the prerequisites on the link you provided. I believe the
first prerequisite is ROS 2 Galactic. Also make sure to go through all the
prerequisites (including the Optional ones). As you go through and find
errors, you can update each piece for ROS 2 Humble.

I am currently writing tutorials for autonomous docking for ROS 2 Jazzy
(latest version of ROS 2). These should be released later this year.

**Comment by Phymin** [comment]
Thanks for your reply, I do find an version issue related with amcl configuration, except that I don't encounter other issues. After read the code, I think the issue may be because in the navigation parameter file, you are still using the default bt tree which will not handle the battery state, but not using the customized bt tree, what do you think?

**Comment by automaticaddison** [comment]
Great! Yes AMCL changed to a new configuration for Humble. Glad you got it
working.

I will not be keeping this package up to date with the latest ROS 2
versions, but I will have a new package for another robot coming out which
I will keep maintained. Thank you!

**Comment by Jianfeng072199** [comment]
Hello, have you finally achieved the automatic docking effect? Can you share it with me? How should this be resolved?

**Comment by Jianfeng072199** [comment]
> Hi
> 
> I follow the tutorial https://automaticaddison.com/auto-docking-to-recharge-battery-aruco-marker-and-ros-2/, but the auto docking function is not working as described, the issue is after run the low_battery_pub.py script, the robot will not plan a path to the staging area, but still moves to the previous goal, am I missing some step here?
> 
> BTW, I am using ros2 humble

Hello, have you finally achieved the automatic docking effect? Can you share it with me? How should this be resolved?

**Comment by automaticaddison** [comment]
Yes. It is all working on ROS 2 Jazzy. Do a search on my blog or YouTube
channel. Thanks!

**Comment by Jianfeng072199** [comment]
Hello, can you share your blog or YouTube channel with me? I couldn't find it

**Comment by automaticaddison** [comment]
https://youtu.be/tFE75bP-ASo?si=7oA9Mfbv81Owt5-X
